pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -944,6 +944,7 @@ func SetDefaults(cr *api.KieApp) {
 		specApply.Objects.Servers = []api.KieServerSet{{Deployments: Pint(constants.DefaultKieDeployments)}}
 	}
 	setKieSetNames(specApply)
+	setJvmDefault(specApply.Objects.Console.Jvm)
```

Comment:
should we also add one for kieserversets since they also have a jvm pointer like console does?
https://github.com/kiegroup/kie-cloud-operator/blob/2253fc607392f8a7f135dde1539832bc22643903/pkg/apis/app/v2/kieapp_types.go#L127

if so, it should likely be added to the for loop below.

Reply:
Ok, I'll add with test